src/Deployment/source.extension.vsixmanifest

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="eb2680f2-4e63-44a8-adf6-2e667d9f689c" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
-    <DisplayName>Roslyn Insiders</DisplayName>
+    <Identity Id="RoslynInsidersWithoutToolWindow" Version="|%CurrentProject%;GetBuildVersion|" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>Roslyn Insiders (Without Tool Window)</DisplayName>
```

Comment:
Is this copy of the extension even useful anymore?  Should we just get rid of it?